src/main/java/nextstep/subway/path/domain/Fare.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.subway.path.domain;
+
+import nextstep.subway.line.domain.Section;
+
+import java.util.List;
+
+public class Fare {
+    private int fare = 1_250;
+
+    public Fare(List<Section> sections, Path path, int age) {
+        calculate(sections, path, age);
+    }
+
+    public static Fare of(List<Section> sections, Path path, int age) {
+        return new Fare(sections, path, age);
+    }
+
+    public void calculate(List<Section> sections, Path path, int age) {
```

Comment:
```suggestion
    private void calculate(List<Section> sections, Path path, int age) {
```
접근제한자를 바꿔주셔도 될 것 같아요.